### ISSUE TITLE
Using  vorpal with wechaty

### ISSUE THREAD

**Issue by shwetalsoni** [wechaty/docusaurus#867]
![testing-windows](https://user-images.githubusercontent.com/57187745/118697237-b6cea580-b82c-11eb-86f1-72601a92f90f.png)

Node version = v12.18.3
vorpal-wechaty = 1.0.0

Above screenshot is for **using vorpal with wechaty** on **WIndows**  and the error it shows up on running npm start.
*It works fine with both Linux and Mac.*
@Huan can you please look into this **"What is causing this error and how it can be eliminated"** so that I can update it in Documentation

**Comment by huan** [comment]
Dear Shwetal,

The error `Navigation timeout of 30000 ms exceed` is caused by puppeteer which means that the webpage has not been loaded corrected.

Normally I believe this error will be resolved by retrying/re-running the code, but if you get the same error serival times then it means there might be some issue with the system.

My suggestion would be you can try it again to see if the problem solved, or you can just focus on writing the feature part of the docs for now, and ignore the windows system for vorpal docs temporary, and we can back to it when we have time to reproduce it and see what happends later.

BTW: it would be better if we can just copy/paste the text error message in issues instead a screen shot. We should use text as possible as we can because it is searchable and copy/pastiable.

**Comment by huan** [comment]
Dear Shwetal,

The error `Navigation timeout of 30000 ms exceed` is caused by puppeteer which means that the webpage has not been loaded corrected.

Normally I believe this error will be resolved by retrying/re-running the code, but if you get the same error serival times then it means there might be some issue with the system.

My suggestion would be you can try it again to see if the problem solved, or you can just focus on writing the feature part of the docs for now, and ignore the windows system for vorpal docs temporary, and we can back to it when we have time to reproduce it and see what happends later.

BTW: it would be better if we can just copy/paste the text error message in issues instead a screen shot. We should use text as possible as we can because it is searchable and copy/pastiable.

**Comment by shwetalsoni** [comment]
Yeah I tried it many times and still it showed the same error but works
fine for other os. So i guess then this is probably the problem with system
.

Thanks Huan :)

**Comment by huan** [comment]
You are welcome!

**Comment by sbis04** [comment]
Hi @shwetalsoni today I tried on a Windows system by doing a clean install (that is none of the build tools were present on the system previously). I have just followed and run the exact steps mentioned in the [Getting started](https://wechaty.js.org/docs/tutorials/using-vorpal-with-wechaty/#getting-started) guide of the Vorpal bot tutorial page, and the bot was running perfectly.

![wechaty_ss_1](https://user-images.githubusercontent.com/43280874/119227619-6ad17880-bb2c-11eb-9f6e-a3a5097e1b47.png)

Have used:
```
WECHATY_PUPPET=wechaty-puppet-whatsapp
```
**node version: v14.17.0**

Try one thing, install the latest version of `node` and then follow the steps.
